docker/docker-compose.yml

```diff
@@ -34,32 +34,46 @@ services:
 
   instance13:
     image: postgres:13
+    command: ["-c", "shared_preload_libraries=pg_stat_statements"]
+    ports:
+      - 5433:5432
```

Comment:
There's no need to change this. Same for other instances. Can you please remove this change from your second commit?

Reply:
Pas sûr de comprendre... tu parles des ports ou de la commande ?